VCFv4.3.tex

```diff
@@ -544,13 +544,14 @@ \subsection{VCF tag naming conventions}
 \begin{itemize}
     \item The `L' suffix means \emph{likelihood} as log-likelihood in the sampling distribution, $\log_{10} \Pr(\mathrm{Data}|\mathrm{Model})$.
     Likelihoods are represented as $\log_{10}$ scale, thus they are negative numbers (e.g.\ GL, CNL).
-    The likelihood can be also represented in some cases as phred-scale in a separate tag (e.g.\ PL).
+    The likelihood can be also represented in some cases as a phred-true scale ($-10 \log_{10}(probability\_of\_being\_correct)$) in a separate tag (e.g.\ PL).
```

Comment:
Do we really want to introduce the term "phred-true scale"?  It seems like a bit of a strange one off.  It's not clear that it's contrasting probability of correct with probability of incorrect until you read the  'Q Suffix' box below.

Reply:
Not really, but I'm unsure how best to describe it given the "P" prefix was almost certainly chosen for Phred.

Right now it states "Likelihoods are represented as $\log_{10}$ scale, thus they are negative numbers".  The intent of the phred-ish scale appears to be to negate that and have a positive scale, so maybe this is the correct way of describing it.  I'll have a stab at doing something more.